storage/innobase/xtrabackup/doc/source/_res/replace/proper.txt

```diff
@@ -1,4 +1,4 @@
-.. Company names (e.g. Percona, Oracle), product names (e.g. MySQL, Ubuntu)
+.. Company names (for example Percona, Oracle), product names (for example MySQL, Ubuntu)
```

Comment:
for example, Percona, Oracle
for example, MySQL, Ubuntu

Reply:
Internal doc - won't fix.